pkgs/development/python-modules/astropy/default.nix

```diff
@@ -23,7 +24,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ numpy pytest ]; # yes it really has pytest in install_requires
 
-  checkInputs = [ pytest pytest-astropy ];
+  checkInputs = [ pytest pytest-astropy bleach ];
```

Comment:
No dont need to add `bleach` for the moment, since the tests are disabled anyway.